code/modules/materials/definitions/solids/materials_solid_mineral.dm

```diff
@@ -236,18 +236,17 @@
 
 /decl/material/solid/clay
 	name = "clay"
+	codex_name = "raw clay"
 	uid = "solid_clay"
-	color = COLOR_OFF_WHITE
+	color = "#807f7a"
 	ore_name = "clay"
-	ore_icon_overlay = "lump"
-	heating_products = list(/decl/material/solid/stone/ceramic = 1)
-	heating_point = GENERIC_SMELTING_HEAT_POINT
-	heating_sound = null
-	heating_message = null
 	ore_compresses_to = null
-	ore_icon_overlay = "dust"
+	ore_icon_overlay = "lump"
 	value = 0.8
 	default_solid_form = /obj/item/stack/material/lump
+	bakes_into_material = /decl/material/solid/stone/pottery
+	melting_point = null // Clay is already almost a liquid...
+	bakes_into_at_temperature = 1330 // roughly the temperature expected from a kiln
```

Comment:
non-blocking but i would personally round this to either 1050, 1060, or 1100C to be honest, 1056.85C is a bit odd

Reply:
@MistakeNot4892 my point here was to use the CELSIUS helper, not to change it to 1300K or 1500K x_x

if you're fine with it just being some weird non-round number when converted to celsius that's fine though, i can approve as-is